extern/storage-sealing/states_sealing.go

```diff
@@ -274,7 +274,7 @@ func (m *Sealing) preCommitParams(ctx statemachine.Context, sector SectorInfo) (
 
 	msd := policy.GetMaxProveCommitDuration(actors.VersionForNetwork(nv), sector.SectorType)
 
-	if minExpiration := height + msd + miner.MinSectorExpiration + 10; expiration < minExpiration {
+	if minExpiration := sector.TicketEpoch + policy.MaxPreCommitRandomnessLookback + msd + miner.MinSectorExpiration; expiration < minExpiration {
```

Comment:
Do we need the `+ msd` here? I'd think that `sector.TicketEpoch + policy.MaxPreCommitRandomnessLookback` covers the entire period in which the sector can be procommitted.

Reply:
Yes. When PreCommit sector [validateExpiration](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/miner/miner_actor.go#L721) in specs-actors, it uses `curEpoch +  MaxProveCommitDuration[precommit.SealProof]` as its activation, so I think we'd better add msd here. `sector.TicketEpoch + policy.MaxPreCommitRandomnessLookback` only covers precommit expire time